exporters/otlp/otlp_test.go

```diff
@@ -338,3 +343,72 @@ func TestSplitDriverFail(t *testing.T) {
 		}
 	}
 }
+
+func ExampleNewExporter() {
+	ctx := context.Background()
+
+	// Set different endpoints for the metrics and traces collectors
+	metricsDriver := otlpgrpc.NewDriver(
+	//Configure metrics driver here
+	)
+	tracesDriver := otlpgrpc.NewDriver(
+	//Configure traces driver here
+	)
+	config := otlp.SplitConfig{
+		ForMetrics: metricsDriver,
+		ForTraces:  tracesDriver,
+	}
+	driver := otlp.NewSplitDriver(config)
+	exporter, err := otlp.NewExporter(ctx, driver) // Configure as needed.
+	if err != nil {
+		log.Fatalf("failed to create exporter: %v", err)
+	}
+	defer func() {
+		err := exporter.Shutdown(ctx)
+		if err != nil {
+			log.Fatalf("failed to stop exporter: %v", err)
+		}
+	}()
+
+	tracerProvider := sdktrace.NewTracerProvider(sdktrace.WithBatcher(exporter))
+	otel.SetTracerProvider(tracerProvider)
+
+	// Output:
+}
+
+func ExampleNewExporter_with_batcher() {
```

Comment:
This tests seems a bit redundant as the previous one also uses the exporter as a batcher. We can probably just unify on the first example as it highlights the full setup process and this only explains how a batching SpanProcessor can be configured which is unrelated to the NewExporter function itself.